lib/msprime.c

```diff
@@ -4363,7 +4356,7 @@ msp_pedigree_climb(msp_t *self)
                 if (u[j] == NULL) {
                     continue;
                 }
-                /* assert(u[j]->prev == NULL); */
+                /* tsk_bug_assert(u[j]->prev == NULL); */
```

Comment:
This looks like a case for a plain `assert`?

Reply:
Good call.